storage/backup/backup-cli/src/metadata/cache.rs

```diff
@@ -202,7 +220,18 @@ pub async fn sync_and_load(
         "Metadata cache loaded.",
     );
 
-    Ok(MetadataView::new(metadata_vec, remote_file_handles))
+    let mut metaview = MetadataView::new(metadata_vec);
+    // initialize if the compaction timestamps is empty
+    if metaview.get_compaction_timestamps().is_empty() {
+        // create a map of files with compaction time set to now as initialization
+        let now = duration_since_epoch().as_secs();
+        let timestampes = remote_file_handles
+            .iter()
+            .map(|e| (e.clone(), now))
+            .collect();
+        metaview.update_compaction_timestamps(timestampes);
+    }
```

Comment:
Do we actually need this logic? After the compaction, all files will be replaced hence tagged now anyway, right?

Reply:
The compaction_timestamps is also used to generate the list of metadata files used for creating this view. 
For the 1st run, if we don't initialize, the file list would be empty causing errors in obtaining the list of files in backup storage.

Reply:
update to none